submissions/kasionio/html-css-cinema-booking/index.html

```diff
@@ -0,0 +1,155 @@
+                     <text x="128" y="40" class="screen_text" fill="white">Screen</text>
+                  </svg>
+                  <div class="seats" id='row-1'>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-1_1' value="1-1">
+                     <label for="seat-1_1" class="label"></label>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-1_2' value="1-2">
+                     <label for="seat-1_2" class="label"></label>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-1_3' value="1-3">
+                     <label for="seat-1_3" class="label"></label>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-1_4' value="1-4" disabled>
+                     <label for="seat-1_4" class="label"></label>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-1_5' value="1-5">
+                     <label for="seat-1_5" class="label"></label>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-1_6' value="1-6">
+                     <label for="seat-1_6" class="label"></label>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-1_7' value="1-7">
+                     <label for="seat-1_7" class="label"></label>
+                  </div>
+                  <div class="seats" id='row-2'>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-2_1' value="2-1">
+                     <label for="seat-2_1" class="label"></label>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-2_2' value="2-2">
+                     <label for="seat-2_2" class="label"></label>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-2_3' value="2-3" disabled>
+                     <label for="seat-2_3" class="label"></label>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-2_4' value="2-4" disabled>
+                     <label for="seat-2_4" class="label"></label>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-2_5' value="2-5" disabled>
+                     <label for="seat-2_5" class="label"></label>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-2_6' value="2-6">
+                     <label for="seat-2_6" class="label"></label>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-2_7' value="2-7">
+                     <label for="seat-2_7" class="label"></label>
+                  </div>
+                  <div class="seats" id='row-3'>
+                     <input class="seat visually-hidden" type="checkbox" name="seat" id='seat-3_1' value="3-1">
```

Comment:
Add styles on `:hover`, and `:active` states. Also, add `cursor: pointer`.

Reply:
Done. Thanks for the review!